src/main/java/com/aws/greengrass/status/model/FleetStatusDetails.java

```diff
@@ -54,10 +54,14 @@ public void setVariablePayload(List<ComponentStatusDetails> variablePayload) {
     }
 
     @Override
+    @SuppressWarnings("PMD.NullAssignment")
     public void setChunkInfo(int chunkId, int totalChunks) {
         if (this.messageType == MessageType.COMPLETE && totalChunks > 1) {
             // set chunk info only if it's a complete update and the message splits into multiple chunks
             chunkInfo = new ChunkInfo(chunkId, totalChunks);
+        } else {
+            // reset chunk info to null to exclude from upload
+            chunkInfo = null;
```

Comment:
this happens if the message type isn't `complete` is that correct?

If there is only 1 chunk, then omitting this seems fine, but if the message type isn't complete but there are multiple chunks, this would still seem necessary.

Reply:
Will update. Also double checked with cloud team that it's not breaking their service.